build/tasks/build.js

```diff
@@ -13,6 +13,14 @@ gulp.task('build-html', function () {
     .pipe(gulp.dest(paths.output + 'system'));
 });
 
+gulp.task('build-css', function () {
```

Comment:
As of bootstrap v3.3.1 .top-left, .top-right, .bottom-left, and .bottom-right classes are deprecated.
So, I thought we might have to ship minor css patches when the above mentioned classes are removed from bootstrap.